README.md

```diff
@@ -26,21 +26,21 @@ ingress:
   enabled: false
   hosts:
     - service.{{ssm "/exists/subdomain" }}
-    - service1.{{ssm "/empty/subdomain" "required=false" }}
-    - service2.{{ssm "/exists/subdomain" "required=false" "region=eu-west-1" }}
-    - service3.{{ssm "/subdomain" "required=false" "region=eu-west-1" "prefix=/empty" }}
+    - service1.{{ssm "/empty/subdomain" "default=codacy.org" }}
+    - service2.{{ssm "/exists/subdomain" "default=codacy.org" "region=eu-west-1" }}
+    - service3.{{ssm "/subdomain" "default=codacy.org" "region=eu-west-1" "prefix=/empty" }}
     - service4.{{ssm "/securestring" }}
 
 ```
 
-when using `required=false` option you might want to disable the key at all if there is no value associated, an easy way of doing it with go templates is:
+when you do not want a key to be defined, use a with in the go templates:
 
 ```yaml
 service:
 ingress:
   enabled: false
   hosts:
-    {{with $subdomain := (ssm "/exists/subdomain" "required=false") }}{{ if $subdomain }}
+    {{with $subdomain := (ssm "/exists/subdomain" "default=") }}{{ if $subdomain }}
```

Comment:
`default=false`
is maybe more clear

Reply:
Since it is a string it will not work, since `"false"` will be truthful

Reply:
right, please detail that only string values are supported in the readme.